src/Domain/Fixture/SubextensionManager.php

```diff
@@ -180,6 +181,30 @@ public function getByParent(Package $package): array {
     return $subextensions;
   }
 
+  /**
+   * Finds the dev-dependencies of a given package.
+   *
+   * @param \Acquia\Orca\Domain\Package\Package $package
+   *   The package to search for dev-dependencies.
+   *
+   * @return array|null
+   *   An indexed array of all the dev-dependencies.
+   *
+   * @throws \Acquia\Orca\Exception\OrcaException
+   * @throws \Acquia\Orca\Exception\OrcaFileNotFoundException
+   * @throws \Acquia\Orca\Exception\OrcaParseError
+   */
+  public function findDevDependenciesByPackage(Package $package): ?array {
```

Comment:
If you _always_ return an array--even an empty one--then code that uses this function can safely just iterate over it without have to check first that it's an array. For example, the code that calls this above:

```diff
- if ($dev_dependencies === NULL || $dev_dependencies === []) {
+ if ($dev_dependencies === []) {
```

Reply:
Done